Common subdirectories: cl-sdl.orig/CVS and cl-sdl/CVS
Common subdirectories: cl-sdl.orig/build and cl-sdl/build
Common subdirectories: cl-sdl.orig/c and cl-sdl/c
Common subdirectories: cl-sdl.orig/debian and cl-sdl/debian
Common subdirectories: cl-sdl.orig/doc and cl-sdl/doc
Common subdirectories: cl-sdl.orig/examples and cl-sdl/examples
Common subdirectories: cl-sdl.orig/ffi and cl-sdl/ffi
Common subdirectories: cl-sdl.orig/nehe and cl-sdl/nehe
Common subdirectories: cl-sdl.orig/opengl and cl-sdl/opengl
diff -u cl-sdl.orig/opengl.asd cl-sdl/opengl.asd
--- cl-sdl.orig/opengl.asd	2002-12-08 09:55:16.000000000 -0600
+++ cl-sdl/opengl.asd	2003-10-17 13:06:13.490855184 -0500
@@ -4,7 +4,7 @@
 
 (in-package #:asdf)
 
-(defsystem opengl
+#-clisp (defsystem opengl
     :components ((:module :opengl
                           :components
                           ((:file "gl")
Common subdirectories: cl-sdl.orig/sdl and cl-sdl/sdl
diff -u cl-sdl.orig/sdl-demos.asd cl-sdl/sdl-demos.asd
--- cl-sdl.orig/sdl-demos.asd	2002-12-08 13:40:44.000000000 -0600
+++ cl-sdl/sdl-demos.asd	2003-10-17 13:06:23.859278944 -0500
@@ -2,7 +2,7 @@
 
 (in-package #:asdf)
 
-(defsystem sdl-demos
+#-clisp (defsystem sdl-demos
     :components ((:module :examples
                           :components
                           ((:file "sdl-test")
diff -u cl-sdl.orig/sdl-examples.asd cl-sdl/sdl-examples.asd
--- cl-sdl.orig/sdl-examples.asd	2002-12-08 10:56:51.000000000 -0600
+++ cl-sdl/sdl-examples.asd	2003-10-17 13:06:31.087180136 -0500
@@ -3,7 +3,7 @@
 ;;;; ASDF System Definition
 (in-package #:asdf)
 
-(defsystem sdl-examples
+#-clisp (defsystem sdl-examples
     :components ((:module :examples
                           :components
                           ((:file "2d-test"
diff -u cl-sdl.orig/sdl-ffi.asd cl-sdl/sdl-ffi.asd
--- cl-sdl.orig/sdl-ffi.asd	2002-12-08 08:48:10.000000000 -0600
+++ cl-sdl/sdl-ffi.asd	2003-10-17 13:06:37.926140456 -0500
@@ -4,7 +4,7 @@
 
 (in-package #:asdf)
 
-(defsystem sdl-ffi
+#-clisp (defsystem sdl-ffi
     :components ((:module :ffi
                           :components ((:file "uffi"))))
     :depends-on (uffi))
Common subdirectories: cl-sdl.orig/sdl-img and cl-sdl/sdl-img
diff -u cl-sdl.orig/sdl-img.asd cl-sdl/sdl-img.asd
--- cl-sdl.orig/sdl-img.asd	2002-12-10 17:27:56.000000000 -0600
+++ cl-sdl/sdl-img.asd	2003-10-17 13:06:44.368161120 -0500
@@ -3,7 +3,7 @@
 ;;;; ASDF System Definition
 (in-package #:asdf)
 
-(defsystem sdl-img
+#-clisp (defsystem sdl-img
     :components ((:module :sdl-img
                           :components ((:file "sdl-img"
                                               :depends-on ("lib"))
Common subdirectories: cl-sdl.orig/sdl-mix and cl-sdl/sdl-mix
diff -u cl-sdl.orig/sdl-mix.asd cl-sdl/sdl-mix.asd
--- cl-sdl.orig/sdl-mix.asd	2002-12-10 17:27:56.000000000 -0600
+++ cl-sdl/sdl-mix.asd	2003-10-17 13:06:50.749191056 -0500
@@ -3,7 +3,7 @@
 ;;;; ASDF System Definition
 (in-package #:asdf)
 
-(defsystem sdl-mix
+#-clisp (defsystem sdl-mix
     :components ((:module :sdl-mix
                           :components ((:file "sdl-mix"
                                               :depends-on ("lib"))
Common subdirectories: cl-sdl.orig/sdl-ttf and cl-sdl/sdl-ttf
diff -u cl-sdl.orig/sdl-ttf.asd cl-sdl/sdl-ttf.asd
--- cl-sdl.orig/sdl-ttf.asd	2002-12-18 18:43:30.000000000 -0600
+++ cl-sdl/sdl-ttf.asd	2003-10-17 13:06:56.721283160 -0500
@@ -3,7 +3,7 @@
 ;;;; ASDF System Definition
 (in-package #:asdf)
 
-(defsystem sdl-ttf
+#-clisp (defsystem sdl-ttf
     :components ((:module :sdl-ttf
                           :components ((:file "sdl-ttf"
                                               :depends-on ("lib"))
diff -u cl-sdl.orig/sdl.asd cl-sdl/sdl.asd
--- cl-sdl.orig/sdl.asd	2002-12-18 18:43:30.000000000 -0600
+++ cl-sdl/sdl.asd	2003-10-17 13:07:02.305434240 -0500
@@ -4,7 +4,7 @@
 
 (in-package #:asdf)
 
-(defsystem sdl
+#-clisp (defsystem sdl
     :components ((:module :sdl
                           :components
                           ((:file "sdl")
